service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/EncounterMapper.java

```diff
@@ -41,6 +49,10 @@
 
     private LocationMapper locationMapper;
 
+    private PatientMapper patientMapper;
```

Comment:
Patient mapper is defined twice which cause compilation error

Reply:
This error was caused by merge conflict.